terraform/environments/prd/aws/main.tf

```diff
@@ -42,13 +42,24 @@ module "vpcs" {
   nat_gws      = local.vpcs["ivanoff-tf"].nat_gws
   subnets      = local.vpcs["ivanoff-tf"].subnets
 }
+
+output "vpc_private_subnet_ids" {
```

Comment:
why do we need to ouput private  subnets in root module?

Reply:
For that they are avaible to other modules

Reply:
and for see password in output

Reply:
what password are we talking about if this PR is about VPC

Reply:
>For that they are avaible to other modules

What modules?

Reply:
> what password are we talking about if this PR is about VPC

thought about one thing wrote about another) 
Output variables allow data created and managed within a module to be accessed from outside that module. export information about private and public subnets created in the VPC module.
or pass it to other modules